### PR TITLE
AAP-8221 - Refining Planning Guide structure

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR addresses the changes required by https://issues.redhat.com/browse/AAP-8221.

Changes include:

- Restructuring the Guide from 1 large Chapter to 8 Chapters.
- Adding intro text where missing.

Preview link (VPN required): http://file.rdu.redhat.com/~ddacosta/AAP8221/master.html

### CODE REVIEW

**Comment by ariordan-redhat** [comment]
@dcdacosta I couldn't build this PR with bccutil. 
I opened a [PR](https://github.com/dcdacosta/red-hat-ansible-automation-platform-documentation/pull/1) against your topic branch with changes that I made to get bccutil working locally.

File: downstream/assemblies/platform/assembly-aap-platform-components.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+include::platform/con-about-automation-hub.adoc[leveloffset=+1]
+
+include::platform/con-about-pa-hub.adoc[leveloffset=+1]
+
+include::platform/con-about-automation-controller.adoc[leveloffset=+1]
+
+include::platform/con-about-services-catalog.adoc[leveloffset=+1]
+
+include::platform/con-overview-automation-mesh.adoc[leveloffet=+1]
```
Comment (ariordan-redhat) [inline downstream/assemblies/platform/assembly-aap-platform-components.adoc:21]:
```suggestion
include::platform/con-overview-automation-mesh.adoc[leveloffset=+1]
```

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
Typo in `leveloffset`; otherwise, LGTM

**Review by lmalivert — APPROVED** [review]

File: downstream/assemblies/platform/assembly-aap-platform-components.adoc (hunk 1 above)
Comment (dcdacosta) [reply downstream/assemblies/platform/assembly-aap-platform-components.adoc]: Thank you! Fixed.

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**